pkg/querier/tripperware/subquery.go

```diff
@@ -0,0 +1,43 @@
+package tripperware
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/prometheus/prometheus/promql/parser"
+	"github.com/weaveworks/common/httpgrpc"
+)
+
+var (
+	ErrSubQueryStepTooSmall = "exceeded maximum resolution of %d points per timeseries in subquery. Try decreasing the step size of your subquery"
```

Comment:
I think you meant to recommend "increasing" the step size? or decreasing the "number of steps"?

Reply:
Good catch. I updated it to `increasing the step size`.